packages/fx-core/templates/plugins/resource/aad/bicep/parameters.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Only include the parameters part, otherwise it's hard to merge parameters from each resource plugin. By the way, I updated the interface to ask each resource plugin return a JSON object. I can help modify related logic in AAD plugin later.

Reply:
Fixed.